openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -170,7 +170,7 @@ const TeamDetailsV1 = ({
     TitleBreadcrumbProps['titleLinks']
   >([]);
   const [addAttribute, setAddAttribute] = useState<AddAttribute>();
-  const [loading, setLoading] = useState<boolean>(true);
+  const [loading, setLoading] = useState<boolean>(false);
```

Comment:
@chirag-madlani, Why we are changing the default initial state?

Reply:
```
const fetchPermissions = async () => {
    setLoading(true);
    try {
      const perms = await getEntityPermission(
        ResourceEntity.TEAM,
        currentTeam.id
      );
      setEntityPermissions(perms);
    } catch (error) {
      showErrorToast(
        error as AxiosError,
        t('message.entity-fetch-error', {
          entity: 'User Permissions',
        })
      );
    } finally {
      setLoading(false);
    }
  };

  useEffect(() => {
    !isEmpty(currentTeam) && fetchPermissions();
  }, [currentTeam]);
```

As you can see above loading will be false only when fetchPermissions is called but is currentTeam is empty it will be end up in forever loading which isn't a good UX
That's why I changes it to default false